doc/man/scons.xml

```diff
@@ -6703,23 +6706,66 @@ to enclose a Python expression to be evaluated.
 See <xref linkend='python_code_substitution'/> below
 for a description.</para>
 
-<para>A variable name
-may also be a Python function
-associated with a
-&consvar; in the environment.
-The function should
-accept four arguments:
+<para>The special pseudo-variables
+<emphasis role="bold">$(</emphasis>
+and
+<emphasis role="bold">$)</emphasis>
+may be used to surround parts of a command line
+that may change
+<emphasis>without</emphasis>
+causing a rebuild--that is,
+which are not included in the signature
+of target files built with this command.
+All text between
+<emphasis role="bold">$(</emphasis>
+and
+<emphasis role="bold">$)</emphasis>
+will be removed from the command line
+before it is added to file signatures,
```

Comment:
action signature..

Reply:
so you're commenting on something I didn't write/change, just moved? sigh...  :)

Reply:
gotcha.

Reply:
it is a good catch, I should have spotted it, but just did yank-put without looking. Will fix.

Reply:
updated

Reply:
hmmm, now some places say build signature and some say action signature. I assume we should strive for consistency?

Reply:
I guess build signature is fine. though slightly less accurate.